feature/SM-6647-Error when granting second CR with Duration Challenge (#660)
departmentfortransport/street-manager-docs: docs/api-documentation/V1.25.10/changelog.md

This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
{: .govuk-body}

Version 1.25.10 (04/02/2021):
{: .govuk-heading-s}
Updated Work API with the following changes:
{: .govuk-body}
<ol class="govuk-list govuk-list--bullet">
  <li>

>     SM-6647: Fixed a bug with <code>PUT /works​/{workReferenceNumber}​/permits​/{permitReferenceNumber}​/alterations​/{permitAlterationReferenceNumber}​/status</code> endpoint,  <code>reasonable_period_end_date</code> now used in date validation when granted_with_duration_challenge is used.

Reviewer: Would wrap the second property in the code tags too for consistency. Same in the latest changelog below.

```suggestion
    SM-6647: Fixed a bug with <code>PUT /works​/{workReferenceNumber}​/permits​/{permitReferenceNumber}​/alterations​/{permitAlterationReferenceNumber}​/status</code> endpoint, <code>reasonable_period_end_date</code> now used in date validation when <code>granted_with_duration_challenge</code> is used.
```

Reviewer 2: I think we could word this is a way that better describes the actual bug scenario. Might not be clear to users what has been fixed here. Maybe something like:
```suggestion
    SM-6647: Fixed a bug with <code>PUT /works​/{workReferenceNumber}​/permits​/{permitReferenceNumber}​/alterations​/{permitAlterationReferenceNumber}​/status</code> endpoint,  <code>reasonable_period_end_date</code> now used in date validation when granted_with_duration_challenge so that when multiple alterations exist a user can enter the original reasonable period end date again.
```

>     SM-6647: Fixed a bug with <code>PUT /works​/{workReferenceNumber}​/permits​/{permitReferenceNumber}​/alterations​/{permitAlterationReferenceNumber}​/status</code> endpoint,  <code>reasonable_period_end_date</code> now used in date validation when <code>granted_with_duration_challenge<code> so that when multiple alterations exist a HA user can enter the original reasonable period end date again.

Reviewer 2: ```suggestion
    SM-6647: Fixed a bug with <code>PUT /works​/{workReferenceNumber}​/permits​/{permitReferenceNumber}​/alterations​/{permitAlterationReferenceNumber}​/status</code> endpoint, <code>reasonable_period_end_date</code> now used in date validation when <code>granted_with_duration_challenge<code> so that when multiple alterations exist a HA user can enter the original reasonable period end date again.
```
